cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs

```diff
@@ -434,3 +460,33 @@ impl<
 		fee
 	}
 }
+
+pub struct XcmFeeManagerFromComponentsBridgeHub<WaivedLocations, HandleFee>(
+	PhantomData<(WaivedLocations, HandleFee)>,
+);
+impl<WaivedLocations: Contains<MultiLocation>, FeeHandler: HandleFee> FeeManager
+	for XcmFeeManagerFromComponentsBridgeHub<WaivedLocations, FeeHandler>
+{
+	fn is_waived(origin: Option<&MultiLocation>, fee_reason: FeeReason) -> bool {
+		let Some(loc) = origin else { return false };
+		if let Export { network: Ethereum { chain_id: 15 }, destination: Here } = fee_reason {
```

Comment:
@claravanstaden 
I saw this `chain: 15` also on other place, we should move this to some constant on one place

Reply:
@bkontur that's a good idea. The chain ID is used in `bridge-hub-rococo` and `asset-hub-rococo`. Where would a good place be to configure this common value?

Reply:
An idea is to add a `snowbridge-rococo-common` crate in our monorepo, under `parachain/runtime`

The `bridges` folder in the SDK repo is very specific to the Polkadot-Kusama bridge.